core/src/Revolution/Processors/Workspace/Lexicon/ReloadFromBase.php

```diff
@@ -41,42 +42,53 @@ public function getLanguageTopics()
      */
     public function process()
     {
-        $language = 'en';
-        $namespace = 'core';
-        $corePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH);
+        $namespace = htmlentities($_POST['namespace']);
+        $language = htmlentities($_POST['language']);
+        $topic = htmlentities($_POST['topicLexicon']);
```

Comment:
If taking my suggestion re variable renaming in the JS file:
```suggestion
        $topic = htmlentities($_POST['lexiconTopic']);
```